databases/sql_snippets_postgres.rst

```diff
@@ -4,52 +4,39 @@ SQL Snippets (Postgres)
 Show Queries Running in Postgres
 --------------------------------
 
-show all queries:
+Show all Queries
```

Comment:
No one really knows, there isn't an authority that comes up with official rules for English (unlike other languages like German). As general rule, first and last words are capitalized and verbs, nouns, pronouns, adverbs and adjectives.